src/core_modules/capture-core/components/List/OnlineList/OnlineList.component.js

```diff
@@ -262,6 +262,7 @@ class Index extends React.Component<Props> {
                             return (
                                 <Row
                                     key={row[rowIdKey]}
+                                    id={row[rowIdKey]}
```

Comment:
I guess this is related to my other comment

Reply:
Well, I think my preferred data-test for this would be something like 'online-list-row-[rownr]'. We discussed this some time ago and I think we agreed that this more generic approach would be the better.

In some of the integration tests I'm checking the actual id of the event/tei is the same as what is retrieved from the api. So I don't feel data-test would be the right place to put the eventId/teiId.

Reply:
Cool thanks for explaining the logic behind it. :)